scripts/vm/hypervisor/xenserver/vmops

```diff
@@ -162,7 +162,7 @@ def setLinkLocalIP(session, args):
 
         if result.lower().strip() == "bridge":
             try:
-                cmd = ["brctl", "addbr", brName]
+                cmd = ["ip", "link", "add", "name", brName, "type", "bridge"]
```

Comment:
I suggest you check if this needs to be done for xenserver; the xenserver scripts run only xenserver host (even though they are part of cloudstack-common; cloudstack copies these scripts to XenServer/XCP-ng hosts where they get executed, not on mgmt server). @shwstppr

Reply:
Has this been resolve/attended @shwstppr ?

Reply:
Though both work. I've reverted the change